src/CMakeLists.txt

```diff
@@ -37,11 +37,17 @@ TARGET_INCLUDE_DIRECTORIES(${KHIVALIB} PUBLIC
                            $<INSTALL_INTERFACE:${KHIVA_INSTALL_INC_DIR}>
                            ${ArrayFire_INCLUDE_DIRS}
                            ${Boost_INCLUDE_DIRS})
-
-# Declare compilation libraries
-TARGET_LINK_LIBRARIES(${KHIVALIB}
-                      ${ArrayFire_Unified_LIBRARIES}
+IF(KHIVA_ONLY_CPU_BACKEND)
+    # Declare compilation libraries
+    TARGET_LINK_LIBRARIES(${KHIVALIB}
+                      ${ArrayFire_CPU_LIBRARIES}
```

Comment:
Do you think that using CMake targets is a better approach than using legacy variables taking into account that the projcet reuqires a modern CMake version.
From the AF lib:

>  ``ArrayFire::af``
   Target for the ArrayFire Unified backend.
 ``ArrayFire::afcpu``
   Target for the ArrayFire CPU backend.
 ``ArrayFire::afcuda``
   Target for the ArrayFire CUDA backend.
 ``ArrayFire::afopencl``
   Target for the ArrayFire OpenCL backend.